digestiflow_demux/bases_mask.py

```diff
@@ -0,0 +1,96 @@
+class BaseMaskConfigException(Exception):
+    """Raise if base mask for demux_reads is malformed"""
+
+    pass
+
+
+def split_bases_mask(bases_mask):
+    """Parse a picard style bases mask
+    and return a list of tuples
+    e.g. for paired-end dual-indexed runs
+    [('T', 100), ('B', 8), ('B', 8), ('T', 100)]"""
```

Comment:
Hm, the comment could use a bit more columns ;) Consider simply adding a doctest

https://docs.python.org/3/library/doctest.html

Reply:
Good idea. done.